OpenRA.Mods.Common/Traits/Infantry/TakeCover.cs

```diff
@@ -19,14 +19,15 @@ namespace OpenRA.Mods.Common.Traits
 	[Desc("Make the unit go prone when under attack, in an attempt to reduce damage.")]
 	public class TakeCoverInfo : TurretedInfo
 	{
-		[Desc("How long (in ticks) the actor remains prone.")]
+		[Desc("How long (in ticks) the actor remains prone.",
+			"Negative values mean actor remains prone permanently.")]
 		public readonly int Duration = 100;
 
 		[Desc("Prone movement speed as a percentage of the normal speed.")]
 		public readonly int SpeedModifier = 50;
 
-		[FieldLoader.Require]
-		[Desc("Damage types that trigger prone state. Defined on the warheads.")]
+		[Desc("Damage types that trigger prone state. Defined on the warheads.",
+			"If Duration is negative (permanent), you can leave this empty to trigger prone state immediately.")]
```

Comment:
What is our use case for immediate permanent prone?

Reply:
If a modder wants to trigger permanent prone state through other means than damage (deploy, for example).